phys2bids/physio_obj.py

```diff
@@ -543,6 +552,40 @@ def print_info(self, filename):
 
         LGR.info(info)
 
+    def auto_trigger_selection(self):
+        """
+        Find a trigger index matching the channels with a regular expresion.
+
+        Parameters
+        ----------
+        self
+         Raises
+        ------
+        Exception
+        More than one possible trigger channel was automatically found.
+        Exception
+        No trigger channel automatically found
+        Notes
+        -----
+        Outcome:
+            trigger_idx : int
+                Automatically retrieved trigger index
```

Comment:
There are some spacing issues in this docstring. The main thing is the lack of empty lines between docstring sections.

Reply:
> This looks great. TBH I'd prefer it if `chtrig` accepted a special string value (e.g., `"auto"`) when you want automatic trigger channel detection. However, I think it is totally cool if 0 is used, if folks don't feel the same way. If so, then the main things I noticed are minor issues with the function docstring.

Does someone else have an opinion on the auto option?